pkgs/apps/qcelemental/default.nix

```diff
@@ -1,45 +1,30 @@
-{ buildPythonPackage, lib, fetchPypi
-# Other dependencies
-, cacert
-# Python dependencies
-, numpy
-, pydantic
-, pint
-, networkx
-, pytestrunner
-, pytestcov
-, pytest
+{ buildPythonPackage, lib, fetchPypi, fetchpatch, numpy
+, pydantic, pint,  networkx, pytestrunner, pytestcov, pytest
 } :
 
 buildPythonPackage rec {
   pname = "qcelemental";
-  version = "0.19.0";
+  version = "0.20.0";
 
-  checkInputs = [
-    pytestrunner
-    pytestcov
-    pytest
-  ];
-
-  propagatedBuildInputs = [
-    numpy
-    pydantic
-    pint
-    networkx
-    cacert
-  ];
+  checkInputs = [ pytestrunner pytestcov pytest ];
+  propagatedBuildInputs = [ numpy pydantic pint networkx ];
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1ljxwhiz1689qijjqzmzydn8q963xx7zxizjnjq0vqg5py9pkyc5";
+    sha256 = "141vw36fmacj897q26kq2bl9l0d23lyqjfry6q46aa9087dcs7ni";
   };
 
+  # FIXME: Fixed upstream but not released yet. Nevertheless critical for correct behaviour.
+  # See https://github.com/MolSSI/QCElemental/pull/265
+  patches = [ ./SearchPath.patch ];
```

Comment:
When the upstream fix is merged we can remove `qcelemental` and the patches.

Reply:
It is not in nixpkgs-unstable yet so I've kept it for the moment in the same version as on nixpkgs master?

Reply:
Yes, that is the disadvantage of `nixpkgs-unstable` here. It can take a few days while until the channels advances.